internal/mode/static/state/graph/backend_tls_policy.go

```diff
@@ -88,21 +88,24 @@ func validateBackendTLSPolicy(
 		valid = false
 		conds = append(conds, staticConds.NewBackendTLSPolicyInvalid(fmt.Sprintf("invalid hostname: %s", err.Error())))
 	}
-	if backendTLSPolicy.Spec.TLS.CACertRefs != nil && backendTLSPolicy.Spec.TLS.WellKnownCACerts != nil {
+
+	caCertRefs := backendTLSPolicy.Spec.Validation.CACertificateRefs
+	wellKnownCerts := backendTLSPolicy.Spec.Validation.WellKnownCACertificates
+	if len(caCertRefs) > 0 && wellKnownCerts != nil {
 		valid = false
-		msg := "CACertRefs and WellKnownCACerts are mutually exclusive"
+		msg := "CACertificateRefs and WellKnownCACertificates are mutually exclusive"
 		conds = append(conds, staticConds.NewBackendTLSPolicyInvalid(msg))
-	} else if backendTLSPolicy.Spec.TLS.CACertRefs != nil && len(backendTLSPolicy.Spec.TLS.CACertRefs) > 0 {
+	} else if len(caCertRefs) > 0 {
```

Comment:
I know this is out of the scope of this PR, but I don't know if it was brought up in the initial PR. We're still trying to figure out our approach to validation and if it's worth re-validating non-security risk fields that CEL already validates. We're not doing it with other resources like HTTPRoute. (this topic is open for discussion in the ClientSettingsPolicy PR)

Reply:
I did see that discussion alright. Will I create a ticket to follow up on this when we have honed our validation strategy?

Reply:
Created https://github.com/nginxinc/nginx-gateway-fabric/issues/1984